File: Domain/Medico.Domain/Models/MedicalRecord.cs
using System;
using System.Collections.Generic;

namespace Medico.Domain.Models
{
    public class MedicalRecord : Entity
    {
        public string Notes { get; set; }

        public Patient Patient { get; set; }

        public Guid PatientId { get; set; }

        public DateTime CreateDate { get; set; }

        public Guid PhysicianId { get; set; }

        public string DocumentType { get; set; }

        public bool IncludeNotesInReport { get; set; }

        public string Diagnosis { get; set; }

        public Guid? Assessment { get; set; }

        public string FileName { get; set; }

        public string FileType { get; set; }

        public string Subject { get; set; }

        public IEnumerable<DocumentMedicalRecord> Docs { get; set; }
    }
}

File: Services/Medico.Api/ClientApp/src/app/patientChart/models/medicalRecord.ts
export class MedicalRecord {
  id?: string;
  notes?: string;
  patientId?: string;
  documentType?: string;
  diagnosis?: string;
  createDate?: any;
  physicianId?: string;
  icdCode?: any;
  includeNotesInReport: boolean;
  assessment?: string;
  fileName?: string;
  documents?: any[];
  subject?: string;

  constructor() {
    this.createDate = new Date();
    this.includeNotesInReport = true;
  }
}
